components/badges/badges.css

```diff
@@ -0,0 +1,49 @@
+@import url("../avatar/avatar.css");
+
+.badge-container {
+	display: flex;
+	align-items: baseline;
+}
+
+.badge {
+	position: relative;
+	margin-right: 1.5rem;
+}
+
+.status-badge {
+	width: 18.5%;
+	height: 19.5%;
+	position: absolute;
+	border-radius: 50%;
+	bottom: 8%;
+	right: 16%;
+	border: 1.3px solid #fff;
+}
+
+.online {
+	background-color: #4caf50;
```

Comment:
Code looks clean, just a suggestion to use variables for colors instead of direct hex codes and names of colors so that if in future you want to change any color, you just have to make a change at a single place.